mxabierto/adela
# Cambio de landing

## Changed landing page

## Changes made: 

 * In adela system has an image that you can see on the landing page which is especified in the next file:
 ### app/assets/images/escuadron.jpg

### How to test:

*   Download this source and start up your local environment.
*  In Adela system when you access to it, you must see then image describe in the next link:

*   https://m3x1c0.atlassian.net/browse/ADELADMX-126
